packages/portalnetwork/src/subprotocols/state/state.ts

```diff
@@ -158,7 +158,7 @@ export class StateProtocol extends BaseProtocol {
   }
 
   public stateStore = async (contentKey: string, content: string) => {
```

Comment:
Why is this called `stateStore` instead of just using the `baseProtocol.store` method below it which is just a noop?

Reply:
Nm, went ahead and renamed things.  I looked at the various usage of `stateStore` and didn't see any reason we needed a separate store method.